web/libs/storybook/src/components/ChipInput.scss

```diff
@@ -0,0 +1,96 @@
+@import "../assets/styles/base.scss";
+
+$danger_color: #ff0000;
```

Comment:
might be a prudent to add the danger color to base.scss to make it available to other components as more components get added here

Reply:
replaced with an existing css var